src/components/Listings/Listings.scss

```diff
@@ -0,0 +1,12 @@
+.listing-section {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.listing-card {
+  border: .5em solid black;
```

Comment:
It looks like you're using this style for your `area-card` as well. If you end up having similar "card" styling, try migrating it all to a mixin that can be shared across components.